decidim-proposals/spec/system/filter_proposals_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 require "spec_helper"
 
-describe "Filter Proposals", type: :system do
+describe "Filter Proposals", :slow, type: :system do
```

Comment:
is :slow simply a tag or it changes some behavior? I can't find doc on this

Reply:
It [is defined in Decidim dev](https://github.com/decidim/decidim/blob/e181d7e67bdf915a3a8e58416c683f52346de047/decidim-dev/lib/decidim/dev/test/rspec_support/capybara.rb#L64) and it increases the time that the tests wait for the server response in Capybara.

This was needed for many tests in this part, because the results generation is pretty slow in dev and test environments. So, instead of adding this tag to individual tests based on the failing tests in a specific computer, I've decided to add the tag to all the tests related to proposals filters.

Reply:
oh.. I didn't know about this custom configuration

by the way @jesusdb, you can try adding this tag in your scroll failing test.